pkg/volume/validation/pv_validation.go

```diff
@@ -69,3 +70,13 @@ func ValidatePathNoBacksteps(targetPath string) error {
 
 	return nil
 }
+
+// ValidateVolumeIsConfigMap will check volume is a configMap
+func ValidateVolumeIsConfigMap(volumes []v1.Volume, mountName string) bool {
+	for _, volume := range volumes {
+		if volume.Name == mountName && volume.ConfigMap != nil {
```

Comment:
It would be better to put `volume.ConfigMap != nil` in front, so that it can directly check failure without checking the name every time. In addition, this only verifies the volume name, does it really have anything to do with subpath?

Reply:
thanks for your advison, i will reconsider this part.